### PR TITLE
Price component

### PR DESCRIPTION
**Pull request by sampettersson**
## [APP-89]

- Render price
- Render discount tag

## Checklist

- [x] Dark/light mode verification
- [x] Landscape verification
- [x] iPad verification
- [x] iPod/iPhone 12 mini/iPhone SE verification

[APP-89]: https://hedvig.atlassian.net/browse/APP-89

### CODE REVIEW

File: Projects/hCore/Sources/MonetaryAmount.swift [hunk 1]
```diff
@@ -36,32 +36,58 @@ public extension MonetaryAmount {
 }
```
Comment (tarikstafford) [inline Projects/hCore/Sources/MonetaryAmount.swift:36]:
```suggestion
}prefix func -(a: MonetaryAmount) -> MonetaryAmount {
    return MonetaryAmount(amount: -a.value, currency: a.currency)
}
```

File: Projects/Offer/Sources/Components/Header/DiscountTag.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                let months = percentageDiscount.percentageNumberOfMonths
+                if months == 1 {
+                    subtitleLabel.value = L10n.offerScreenPercentageDiscountBubbleTitleSingular(Int(percentageDiscount.percentageDiscount))
+                } else {
+                    subtitleLabel.value = L10n.offerScreenPercentageDiscountBubbleTitlePlural(Int(percentageDiscount.percentageDiscount), months)
+                }
+            } else if let monthlyDeduction = incentiveFragment?.asMonthlyCostDeduction {
+                titleLabel.value = L10n.offerScreenPercentageDiscountBubbleTitle
+                titleLabel.animationSafeIsHidden = false
+                subtitleLabel.value = "-\(monthlyDeduction.amount?.fragments.monetaryAmountFragment.monetaryAmount.formattedAmount ?? "")\(L10n.perMonth)"
```
Comment (tarikstafford) [inline Projects/Offer/Sources/Components/Header/DiscountTag.swift:86]: maybe `monthlyDeduction.amount?.fragments.monetaryAmountFragment.monetaryAmount.negative.formattedAmount ?? ""` is a bit cleaner? also we could add a `prefix func -` to monetary amount, I think the negative is good enough though.

Comment (tarikstafford) [reply Projects/Offer/Sources/Components/Header/DiscountTag.swift:86]: Screenshot 2021-05-03 at 09 55 36

Comment (tarikstafford) [reply Projects/Offer/Sources/Components/Header/DiscountTag.swift:86]: `Coverage` has some hardcoded strings.

Comment (tarikstafford) [reply Projects/Offer/Sources/Components/Header/DiscountTag.swift:86]: Ah coverage ain't done yet. Is there a ticket for this section?

Comment (sampettersson) [reply Projects/Offer/Sources/Components/Header/DiscountTag.swift:86]: @tarikstafford will actually replace this logic with this https://github.com/HedvigInsurance/giraffe/pull/1169 too much damn code to unwrap these 🤠 & yeah, there's a bit of untranslated stuff right now

Comment (sampettersson) [reply Projects/Offer/Sources/Components/Header/DiscountTag.swift]: @tarikstafford updated now!

**Comment by tarikstafford** [comment]
Screenshot 2021-05-05 at 12 06 55

says kr/man not sek/mo for me?

**Comment by tarikstafford** [comment]
I am going it ignore all the language discrepancies for now.

**Review by tarikstafford — APPROVED** [review]

**Pull request merged**